rest/route/agent.go

```diff
@@ -1671,3 +1671,50 @@ func (h *createGitHubDynamicAccessToken) Run(ctx context.Context) gimlet.Respond
 		Token: token,
 	})
 }
+
+// DELETE /rest/v2/task/{task_id}/github_dynamic_access_token
+// This route is used to revoke user-used GitHub access token for a task.
+type revokeGitHubDynamicAccessToken struct {
+	taskID string
+	token  string
+
+	env evergreen.Environment
+}
+
+func makeRevokeGitHubDynamicAccessToken(env evergreen.Environment) gimlet.RouteHandler {
+	return &revokeGitHubDynamicAccessToken{env: env}
+}
+
+func (h *revokeGitHubDynamicAccessToken) Factory() gimlet.RouteHandler {
+	return &revokeGitHubDynamicAccessToken{env: h.env}
+}
+
+func (h *revokeGitHubDynamicAccessToken) Parse(ctx context.Context, r *http.Request) error {
+	if h.taskID = gimlet.GetVars(r)["task_id"]; h.taskID == "" {
+		return errors.New("missing task_id")
+	}
+
+	body, err := io.ReadAll(r.Body)
+	if err != nil {
+		return errors.Wrap(err, "reading body")
+	}
+
+	err = json.Unmarshal(body, &h.token)
+
+	errorMessage := fmt.Sprintf("reading token body for task '%s'", h.taskID)
```

Comment:
I copied this fron other implementations of routes like [here](https://github.com/ZackarySantana/evergreen/blob/96a91e52a5674b05c4025add2c60f3d06f717d15/rest/route/agent.go#L1310). Should I make a ticket to switch these or not worth the effort (I'll fix this one ofc)?

It would be a quick win to make sure messages are static and any dynamic is just the other message.Fields fields.

Reply:
If it's really quick, then I'm fine if you want to add it here as a drive-by fix. Otherwise, a quick-win ticket would be good.